cmd/buildkitd/config/gcpolicy_unix.go

```diff
@@ -3,4 +3,10 @@
 
 package config
 
-var DiskSpacePercentage int64 = 10
+const (
+	DiskSpaceReservePercentage int64 = 10
+	DiskSpaceReserveBytes      int64 = 10 * 1e9 // 10GB
```

Comment:
`DiskSpaceReservePercentage` -> `DiskSpaceMinPercentage`

I'm also not sure the default should have a minimum expressed as a percentage, I think just 10GB should be enough, but not a blocker.

Reply:
`10GB` currently works as a maximum for reserve space, the cache can still be smaller on a small disk. If we would just leave it as static `10GB` then on a small disk the current 10% based limit will go bigger and start to use more storage, even if there is no free space.